jumpcloud-ADMU/Powershell/Private/DisplayForms/Form.ps1

```diff
@@ -713,6 +792,17 @@ Function Show-SelectionForm {
                     Write-ToLog "User $($tb_JumpCloudUserName.Text) does not have a local account on this system"
                 }
             }
+            if ([string]::IsNullOrEmpty($SelectedUserName)) {
+                $SelectedUserName = $($lvProfileList.SelectedItem.username)
+            }
+
+            # Preload progress form to shrink perceived wait between click and display
+            $profilePath = $lvProfileList.SelectedItem.LocalPath
+            $displayUsername = if ([string]::IsNullOrWhiteSpace($tb_JumpCloudUserName.Text)) { $SelectedUserName } else { $tb_JumpCloudUserName.Text }
+            $script:ProgressBar = New-ProgressForm
+            $FormResults.ProgressBar = $script:ProgressBar
+            Write-ToProgress -form $true -ProgressBar $script:ProgressBar -status "Init" -username $SelectedUserName -newLocalUsername $displayUsername -profileSize "Calculating" -LocalPath $profilePath
```

Comment:
I'm fine with this change "Calculating" is okay with me. This is great, there was no need to wait until start migration to start this.